base_classes/NXdetector_channel.nxdl.xml

```diff
@@ -0,0 +1,103 @@
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition category="base" extends="NXobject" name="NXdetector_channel"
+            type="group"
+            xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
+            xmlns="http://definition.nexusformat.org/nxdl/3.1"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:xs="http://www.w3.org/2001/XMLSchema"
+            xmlns:ns="http://definition.nexusformat.org/nxdl/@NXDL_RELEASE@"
+            >
+
+  <symbols>
+    <doc>
+      These symbols will be used below to illustrate the coordination of the rank and sizes of datasets and the 
+      preferred ordering of the dimensions. Each of these are optional (so the rank of the datasets 
+      will vary according to the situation) and the general ordering principle is slowest to fastest.
+      The type of each dimension should follow the order of scan points, detector output (e.g. pixels),
+      then time-of-flight (i.e. spectroscopy, spectrometry). Note that the output of a detector is not limited 
+      to single values (0D), lists (1D) and images (2), but three or higher dimensional arrays can be produced 
+      by a detector at each trigger.
+    </doc>
+    
+    <symbol name="dataRank"><doc>Rank of the ``data`` field associated with this detector</doc></symbol>
+    <symbol name="nP"><doc>number of scan points</doc></symbol>
+    <symbol name="i"><doc>number of detector pixels in the slowest direction</doc></symbol>
+    <symbol name="j"><doc>number of detector pixels in the second slowest direction</doc></symbol>
+    <symbol name="k"><doc>number of detector pixels in the third slowest direction</doc></symbol>
+  </symbols>
+
+  <doc>
+    Description and metadata for a single channel from a multi-channel detector.
+
+    Given an NXdata group linked as part of an NXdetector group that has an axis with named channels (see the
+    example in NXdata), the NXdetector will have a series of NXdetector_channel groups, one for each
+    channel, named CHANNELNAME_channel.
+
+    Example, given these axes in the NXdata group:
+      @axes = ["image_id", "channel", ".", "."]
+
+    And this list of channels in the NXdata group:
+      channel = ["threshold_1", "threshold_2", "difference"]
+
+    The NXdetector group would have three NXdetector_channel groups:
```

Comment:
```suggestion
    Example, given these axes in the NXdata group::
    
      @axes = ["image_id", "channel", ".", "."]

    And this list of channels in the NXdata group::
    
      channel = ["threshold_1", "threshold_2", "difference"]

    The NXdetector group would have three NXdetector_channel groups::
    
```
Putting these examples in a code block makes it easier to read in my opinion and it matches other parts of the documentation better (for example in NXdata).

The html version then looks like this
![image](https://user-images.githubusercontent.com/73243774/231159001-d2dd5dce-72ad-43d5-af9e-9cf75aceb006.png)